tree/ntuple/v7/src/RColumnElement.cxx

```diff
@@ -67,37 +67,37 @@ std::pair<std::uint16_t, std::uint16_t>
 ROOT::Experimental::Internal::RColumnElementBase::GetValidBitRange(EColumnType type)
 {
    switch (type) {
-   case EColumnType::kIndex64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kIndex32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kSwitch: return std::make_pair<std::uint16_t, std::uint16_t>(96, 96);
-   case EColumnType::kByte: return std::make_pair<std::uint16_t, std::uint16_t>(8, 8);
-   case EColumnType::kChar: return std::make_pair<std::uint16_t, std::uint16_t>(8, 8);
-   case EColumnType::kBit: return std::make_pair<std::uint16_t, std::uint16_t>(1, 1);
-   case EColumnType::kReal64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kReal32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kReal16: return std::make_pair<std::uint16_t, std::uint16_t>(16, 16);
-   case EColumnType::kInt64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kUInt64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kInt32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kUInt32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kInt16: return std::make_pair<std::uint16_t, std::uint16_t>(16, 16);
-   case EColumnType::kUInt16: return std::make_pair<std::uint16_t, std::uint16_t>(16, 16);
-   case EColumnType::kInt8: return std::make_pair<std::uint16_t, std::uint16_t>(8, 8);
-   case EColumnType::kUInt8: return std::make_pair<std::uint16_t, std::uint16_t>(8, 8);
-   case EColumnType::kSplitIndex64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kSplitIndex32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kSplitReal64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kSplitReal32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kSplitInt64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kSplitUInt64: return std::make_pair<std::uint16_t, std::uint16_t>(64, 64);
-   case EColumnType::kSplitInt32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kSplitUInt32: return std::make_pair<std::uint16_t, std::uint16_t>(32, 32);
-   case EColumnType::kSplitInt16: return std::make_pair<std::uint16_t, std::uint16_t>(16, 16);
-   case EColumnType::kSplitUInt16: return std::make_pair<std::uint16_t, std::uint16_t>(16, 16);
+   case EColumnType::kIndex64: return std::make_pair(64, 64);
```

Comment:
I assume this now involves a conversion from `std::pair<int, int>` to `std::pair<std::uint16_t, std::uint16_t>`. Not sure if this is relevant...

Reply:
I checked the generated assembly for gcc, clang and msvc and they all look the same with or without the explicit template arguments (with at least -O1 - they are different in -O0).
Since we're always using constant values that are small I don't think there's going to ever be a practical difference

Reply:
This sounds good but however small of a change, it is still a functional change and I would have recommended for it to be on its own commit (the title of the commit implies that it is strictly a non functional change).